templates/field.php

```diff
@@ -10,7 +10,12 @@
 	<input type="hidden" class="fm-element zone-name" name="<?php echo esc_attr( $this->get_form_name() ); ?>" value="" />
 
 	<div class="zone-search-wrapper">
-		<label for="<?php echo esc_attr( $this->get_element_id() ); ?>_recent"><?php esc_html_e( 'Add Recent Content', 'fm-zones' ); ?></label><br>
+		<label for="<?php echo esc_attr( $this->get_element_id() ); ?>_recent"><?php
+			printf(
+				'%1$s %2$s',
+				esc_html__( 'Add Recent', 'fm-zones' ),
```

Comment:
This should be "Add Recent %s" and the sprintf should wrap that. There should then be an `esc_html()` around that.